### ISSUE TITLE
Release TwistedMail-0.4

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2360]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#2360|
|Type|enhancement|
|Created|2007-01-06 20:40:37Z|

<details><summary>Searchable metadata</summary>

```
trac-id__2360 2360
type__enhancement enhancement
reporter__radix radix
priority__normal normal
milestone__Mail_0_4 Mail-0.4
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1168116037000000 1168116037000000
changetime__1168135064000000 1168135064000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_1"></a>|@radix removed owner|
|-|-|

Ready for review at releases/twisted-mail-0.4.x-2360

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @radix|
|-|-|

"Plaintext logins are now possible over SSL or TLS ([[#1809](https://github.com/twisted/twisted/issues/1809)](https://github.com/twisted/twisted/issues/1809))" should probably say "Plaintext POP3 logins are now possible over SSL or TLS (#1809)"

There's a # missing in "The POP3 client can now correctly deal with concurrent POP3 retrievals ([#1988](https://github.com/twisted/twisted/issues/1988), 1691)".  Also the second "POP3" may be redundant.

Otherwise good.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_3"></a>|@radix set status to `closed`|
|-|-|

(In [19229]) Merge twisted-mail-0.4.x-2360

Author: release automation, radix
Reviewer: exarkun
Fixes [#2360](https://github.com/twisted/twisted/issues/2360)

Version numbers and NEWS from mail 0.4.0

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
